docs/reduction-in-locks.md

```diff
@@ -0,0 +1,75 @@
+
+Percona Server for MySQL implemented `LOCK TABLES FOR BACKUP` to block Data Definition Language (DDL) statements on a server. Percona XtraBackup utilizes this lock during the backup process to ensure that any `DDL` event does not corrupt the backup. The lock does not affect the Data Manipulation Language (DML) statements.
+
+## How the backup lock works
+
+With the [`--lock-ddl=ON`](./xtrabackup-option-reference.md/#lock-ddl) option the backup lock is enabled by default from the start of the backup process.
+
+* Set a backup lock: Set a backup lock on the server to prevent any `DDL` operations, ensuring no new tables are created, renamed, or dropped during the backup.
+
+* Copy the redo logs: Transfer all redo logs from the checkpoint up to the current Log Sequence Number (LSN) and start tracking new entries.
+
+* Identify and copy all InnoDB tablespace, `.ibd`, files. This step takes most of the time and is performed under the lock.
+
+* Copy non-InnoDB files.
+
+* Gather sync point: Collect synchronization points from all engines (InnoDB LSN, binlog, Global Transaction Identifier (GTID), etc.) by querying the log status.
+
+* Stop the redo follow thread: Cease the redo fllow thread once it has copied up to the sync point.
+
+* Release the backup lock on the server.
+
+If the backup lock is disabled with the [`--lock-ddl=OFF`](./xtrabackup-option-reference.md/#lock-ddl) option, a backup continues while concurrent DDL events are executed. These backups may be invalid and may fail at either the `backup` or the `--prepare` step.
+
+## The `--lock-ddl=REDUCED` option overview
+
+[Percona XtraBackup 8.4.0-2](./release-notes/8.4.0-2.md) adds the [`--lock-ddl=REDUCED`](./xtrabackup-option-reference.md/#lock-ddl) option to reduce the time the server remains under the backup lock. Now, you can execute more `DDL` operations concurrently while the backup is in progress. The backup lock is now taken after copying the `.ibd` files and before copying the `non-InnoDB` files.
+
+Copying the InnoDB files with the `--lock-ddl=REDUCED` option is split into two phases.
+
+### Phase 1: `Operations performed without the lock`
+
+1. Copying all redo logs from checkpoint up to the current LSN and start following new entries.
+2. Starting the redo log thread.
+3. Tracking file operations from the redo log.
+4. Copying of all .ibd without taking any lock.
+5. Acquiring Lock Tables For Backup (LTFB)/Lock Instance For Backup (LIFB). This step ensures no new DDL operations, such as creating or altering tables, will occur.
+6. Quering the log_status to discover the LSN. This step ensures that all subsequent operations are tracked and that the backup process captures a consistent state of the database.
+7. Copying all InnoDB files.
+8. Ensuring the redo log has catch up to LSN from step 5.
```

Comment:
I dont have strong opinion, may be @patrickbirch can help us. should we avoid 'ing' at start of every verb ?